src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupService.java

```diff
@@ -2106,4 +2106,191 @@ public static ApiElastigroupGetInstanceStatusResponse getInstanceStatus(String i
 
         return retVal;
     }
+
+    public static ApiElastigroup getBeanstalkConfig(String environmentId, String region, String authToken,
+                                                                            String account) throws SpotinstHttpException {
+        // Init retVal
+        ApiElastigroup retVal = new ApiElastigroup();
+
+        // Get endpoint
+        SpotinstHttpConfig config      = SpotinstHttpContext.getInstance().getConfiguration();
+        String             apiEndpoint = config.getEndpoint();
+
+        Map<String, String> queryParams = new HashMap<>();
+
+        // Add account Id Query param
+        if (account != null) {
+            queryParams.put("accountId", account);
+        }
+        // Add environment Id Query param
+        if (account != null) {
+            queryParams.put("environmentId", environmentId);
+        }
+        // Add region Query param
+        if (account != null) {
+            queryParams.put("region", region);
+        }
+
+        // Get the headers for AWS.
+        Map<String, String> headers = buildHeaders(authToken);
+
+        // Build URI
+        String uri = String.format("%s/aws/ec2/group/beanstalk/import", apiEndpoint);
+
+        // Send the request.
+        RestResponse response = RestClient.sendGet(uri, headers, queryParams);
+
+        // Handle the response.
+        ElastigroupApiResponse getBeanstalkConfigResponse = getCastedResponse(response, ElastigroupApiResponse.class);
```

Comment:
create elasticBeanstalk class (BL and API) to catch the response

Reply:
Done